src/components/brands/BrandTile.jsx

```
@@ -0,0 +1,64 @@
+
+export default function BrandTile({
+  brand,
+  isFollowing,
+  onToggleFollow,
+  onOpen,
+  lastUpdateLabel,
+}) {
+  if (!brand) return null;
+  const name = brand.name || brand.displayName || 'Brand';
+  const handleOpen = () => {
+    try { track('brand_tile_click', { brandId: brand.id }); } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)